doc-content/drools-docs/src/main/asciidoc/DecisionEngine/rule-base-configuration-con.adoc

```diff
@@ -0,0 +1,70 @@
+endif::[]
+
+The following rule base configuration options are available for the {DECISION_ENGINE}:
+
+drools.consequenceExceptionHandler::
+When configured, this property defines the class that manages the exceptions thrown by rule consequences. You can use this property to specify a custom exception handler for rule evaluation in the {DECISION_ENGINE}.
++
+--
+Default value: `org.drools.core.runtime.rule.impl.DefaultConsequenceExceptionHandler`
+
+[source]
+----
+drools.consequenceExceptionHandler=org.drools.core.runtime.rule.impl.MyCustomConsequenceExceptionHandler
+----
+--
+
+drools.multithreadEvaluation::
+When enabled, this property enables the {DECISION_ENGINE} to evaluate rules in parallel by dividing the Rete and Phreak pattern-matching algorithms (if applicable) in independent partitions and evaluating them in parallel. You can use this property if you have retained both Rete and Phreak algorithm functionality in your {PRODUCT} distribution.
++
+--
+Default value: `false`
+
+You can enable multithreaded evaluation using one of the following options:
+
+* Enable multithreaded evaluation with `KieBaseConfiguration`:
++
+[source]
+----
+KieServices ks = KieServices.Factory.get();
+KieBaseConfiguration kieBaseConf = ks.newKieBaseConfiguration();
+kieBaseConf.setOption(MultithreadEvaluationOption.YES);
+KieBase kieBase = kieContainer.newKieBase(kieBaseConf);
+----
+
+* Enable the multithreaded evaluation system property:
++
+[source]
+----
+drools.multithreadEvaluation=true
+----
+
+[WARNING]
+====
+Rules that use queries, salience, or agenda groups are currently not supported by the parallel {DECISION_ENGINE}. If these rule elements are present in the KIE base, the compiler emits a warning and automatically switches back to single-threaded evaluation. However, in some cases, the {DECISION_ENGINE} might not detect the unsupported rule elements and rules might be evaluated incorrectly. For example, the {DECISION_ENGINE} might not detect when rules rely on implicit salience given by rule ordering inside the DRL file, resulting in incorrect evaluation due to the unsupported salience attribute.
+====
+--
+
+drools.sequential::
+When enabled, this property enables sequential mode in the {DECISION_ENGINE}. In sequential mode, the {DECISION_ENGINE} evaluates rules one time in the order that they are listed in the {DECISION_ENGINE} agenda without regard to changes in the working memory. This means that the {DECISION_ENGINE} ignores any `insert`, `modify`, or `update` statements in rules and executes rules in a single sequence. As a result, rule execution may be faster in sequential mode, but important updates may not be applied to your rules. You can use this property if you use stateless KIE sessions and you do not want the execution of rules to influence subsequent rules in the agenda. Sequential mode applies to stateless KIE sessions only.
```

Comment:
Right. We just try to avoid links in module files for single-sourcing and reusability reasons. Can't be helped at times, like in prerequisites, but typically should be avoided.